.github/pull_request_template.md

```diff
@@ -21,7 +21,7 @@
 
 Self checklist (all need to be checked):
 - [ ] The developer has manually tested the changes and verified that the changes work
-- [ ] The commits have meaningful messages (squashes happen on merge by the bot).
+- [ ] The developer has squashed their branch so it contains _only one commit_, with a meaningful message (don't do this if merging a feature branch into main or incubation).
```

Comment:
Can we say "has been squashed meaningfully"? I wonder if we are too strict with the 1 commit methodology. Sometimes multiple commits are good if they are isolated chunks of work. 

* Implemented the table view
* Updated the text
* Review commits
* tmp

These are bad commit messages, because they are effectively "you implemented the table with feedback". But you could have:

* DS Flow support
* Admin Checkbox to enable feature

Something more "these are two different things" sorta deal. I'm not super prescriptive for what we do with our commits, but I think we just need to get rid of those "I'm working on it" sorta commits. Which can be fixed with a `git commit --amend` instead of a `git commit`.

Reply:
I agree with one commit being too strict. How about something like:

"Commits have been squashed into descriptive, self-contained units of work (e.g. any 'WIP' or 'Implements feedback' commit messages have been removed"

Reply:
@andrewballantyne changes made if you'd like to approve.